### DISCUSSION TITLE
Vote on Roadmap

### DISCUSSION THREAD

**Discussion by younes-alouani** [prisma/prisma#14817] [Q&A]
Hello, 

Prisma seems to be an open-source library, isn't it? In that case, its road map should be decided by the user community. 
Wouldn't it be interesting to integrate a voting system for the features to be integrated first ?

**Comment by austincrim** [comment]
Hey @younes-alouani 👋,

All roadmap candidates should be found in GitHub Issues and, therefore, are eligible for upvoting, questions, and lively discussion.

The Prisma team takes all of these factors into account when deciding on what to work on. If you would like to influence our future work, be vocal in GitHub Issues and share them broadly 🙂.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
